release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -238,6 +238,68 @@ In the table, features are marked with the following statuses:
 [id="ocp-4-9-removed-features"]
 === Removed features
 
+==== Beta APIs removed from Kubernetes v1.22
```

Comment:
I'd just say 1.22. We usually don't put the "v" in front. (A few more instances below)